modules/navigation/nav_region.cpp

```diff
@@ -105,10 +105,16 @@ void NavRegion::update_polygons() {
 		return;
 	}
 
+#ifdef DEBUG_ENABLED
 	if (!Math::is_equal_approx(double(map->get_cell_size()), double(mesh->get_cell_size()))) {
 		ERR_PRINT_ONCE("Navigation map synchronization error. Attempted to update a navigation region with a navigation mesh that uses a different `cell_size` than the `cell_size` set on the navigation map.");
 	}
 
+	if (map && Math::rad_to_deg(map->get_up().angle_to(transform.basis.get_column(1))) >= 90.0f) {
+		ERR_PRINT("Navigation map synchronization error. Attempted to update a navigation region transform rotated 90 degree or more away from the current navigation map UP orientation.");
```

Comment:
```suggestion
		ERR_PRINT("Navigation map synchronization error. Attempted to update a navigation region transform rotated 90 degrees or more away from the current navigation map UP orientation.");
```

Reply:
Should it be `ERR_PRINT_ONCE` to avoid spam, like the previous check?

Reply:
Unsure, with print once I had a hard time spotting the error myself when rotating the region in the editor or at runtime.

If the user does not change the region every frame it will not spam as it only check once in the sync phase while the region is dirty.